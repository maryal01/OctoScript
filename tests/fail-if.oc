if (true) {
    "twelve";
}
else {
    12;
}