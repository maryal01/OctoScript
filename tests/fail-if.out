illegal if expression boolean string  int in if true "twelve" else 12